rush.json

```diff
@@ -588,7 +588,14 @@
       "tags": ["tools"],
       "shouldPublish": true,
       "versionPolicyName": "alpha"
-    }
+    },
+
+
+    {
+      "packageName": "common-scripts-precommit",
+      "projectFolder": "common/scripts/pre-commit",
+      "shouldPublish": false
+    },
```

Comment:
Can we rename this to `@kadena-dev/precommit-scripts` or something alike and sort inside tools by packageName?